openfaas/config.yaml

```diff
@@ -0,0 +1,64 @@
+# this will be replaced by Okteto during deployment
+functionNamespace: $NAMESPACE
+serviceType: ClusterIP
+
+# okteto manages the RBAC 
+rbac: false
+
+# disabled by default to make development easier
+basic_auth: false
```

Comment:
Would it be possible to enable it and configure the password here, in the config.yaml, with a default value?

Reply:
we can configure it to have auth enabled, and then the chart creates the secret. but then the user needs to go to the CLI to get the secret before being able to use the gateway. I fee like it's a lot of friction. what do you think @pchico83 ?

Reply:
I think the openfaas gateway shouldnt be open to the world... I think that if the user configures the password in the config-yaml that the Okteto Cloud UI shows (with a default value) is not too much friction